tests/attendee/attendee-repository.php

```diff
@@ -206,6 +221,46 @@ public function test_get_events_for_user() {
 		$this->assertEmpty( $this->repository->get_events_for_user( $another_user + 1 ) );
 	}
 
+	public function test_is_new_translation_contributor() {
+		$this->set_normal_user_as_current();
+		$now      = new DateTimeImmutable( 'now', new DateTimeZone( 'UTC' ) );
+		$user1_id = 52;
+		$user2_id = 53;
+		$user3_id = 54;
+
+		// create 10 translations  for $user2_id before event start.
+		for ( $i = 0; $i < 10; $i++ ) {
+			$this->translation_factory->create( $user2_id );
+
+		}
+		// create 11 translations  for $user3_id before event start.
+		for ( $i = 0; $i < 11; $i++ ) {
+			$this->translation_factory->create( $user3_id );
+		}
+
+		$event1_id  = $this->event_factory->create_active( array(), $now->modify( '+1 day' ) );
+		$attendee11 = new Attendee( $event1_id, $user1_id );
+		$attendee12 = new Attendee( $event1_id, $user2_id );
+		$attendee13 = new Attendee( $event1_id, $user3_id );
+		$this->repository->insert_attendee( $attendee11 );
+		$this->repository->insert_attendee( $attendee12 );
+		$this->repository->insert_attendee( $attendee13 );
+		$event = $this->event_repository->get_event( $event1_id );
+
+		$contributor_ids     = array( $attendee11->user_id(), $attendee12->user_id(), $attendee13->user_id() );
+		$new_contributor_ids = array();
+		$translations_counts = $this->translation_repository->count_translations_before( $contributor_ids, $event->start() );
+		foreach ( $translations_counts as $user_id => $count ) {
+			if ( $count <= 10 ) {
+				$new_contributor_ids[ $user_id ] = true;
+			}
+		}
+
+		$this->assertCount( 2, $new_contributor_ids );
+		$this->assertTrue( $new_contributor_ids[ $user1_id ] );
+		$this->assertTrue( $new_contributor_ids[ $user2_id ] );
```

Comment:
The asserts would then look like this:
```suggestion
		$this->assertTrue( $attendee11->is_new_contributor() );
		$this->assertTrue( $attendee12->is_new_contributor() );
		$this->assertFalse( $attendee13->is_new_contributor() );
```